mobile/lib/services/isolated_ffmpeg_service.dart

```diff
@@ -0,0 +1,12 @@
+import "package:ffmpeg_kit_flutter/ffmpeg_kit.dart";
+import "package:ffmpeg_kit_flutter/ffmpeg_session.dart";
+
+class IsolatedFfmpegService {
+  static Future<FFmpegSession> ffmpegRun(Map args) async {
+    final command = args['command'] as String;
+
+    final session = await FFmpegKit.execute(command);
+
+    return session;
+  }
+}
```

Comment:
Instead of using shared computer/isolate, let's use a dedicated computer instance for ffmpeg so that it doesn't block one of the shared computer for long time.
Also, we can simplify the call by adding a helper method.
class IsolatedFfmpegService {
  final Computer _computer = Computer.create();
  void init() {
    _computer.turnOn(workersCount: 1);
  }
  Future<FFmpegSession> runFfmpeg(String command) async {
    return _computer.compute<Map, FFmpegSession>(
      _ffmpegRun,
      param: {'command': command},
    );
  }

  static Future<FFmpegSession> _ffmpegRun(Map args) async {
    final command = args['command'] as String;
    final session = await FFmpegKit.execute(command);
    return session;
  }
}

Reply:
If you want to init ffmpeg inside that isolate/worker, that can either work with this computer with single worker that will ensure that the ffmpeg is initialized in that isolate or start another isolate and manage it ourself similar to MLComputer